.github/CONTRIBUTNG.md

```diff
@@ -0,0 +1,10 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit helps, and credit will always be given.
+
+Please read the PAVICS [Developer Documentation](https://pavics-sdi.readthedocs.io/en/latest/dev/index.html) and 
+[End-to-End Workflow Test Documentation](https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests#readme) to get started.
+
+Since this repository can be used to test multiple organization's production server, contributions should be backward-compatible as much as possible or feature a toggle so that organizations can activate the new feature when ready without being disturbed in the meantime.
```

Comment:
"or feature a toggle switch" -> or at least be activated via feature toggle

Reply:
We agreed to keep it simple.